src/pages/forgotPassword/VerifyEmail.tsx

```diff
@@ -0,0 +1,95 @@
+import { FunctionComponent, useState } from "react";
+import { useDispatch } from "react-redux";
+import { Box, useTheme } from "@mui/material";
+import { FormikValues, useFormikContext } from "formik";
+import { Button, GradientTypography, Typography } from "elements";
+import { TextInputField } from "components";
+import { sendVerificationEmail } from "modules/session";
+
+const VerifyEmail: FunctionComponent = () => {
+  const dispatch = useDispatch();
+  const theme = useTheme();
+  const [showResendLink, setShowResendLink] = useState(true);
+  const {
+    values: { email },
+  } = useFormikContext<FormikValues>();
+
+  const handleEmailResend = () => {
+    setShowResendLink(false);
+
+    dispatch(sendVerificationEmail(email));
+
+    setTimeout(() => setShowResendLink(true), 10000);
+  };
+
+  return (
+    <Box
+      sx={ {
+        alignItems: "center",
+        display: "flex",
+        flexDirection: "column",
+        height: "100%",
+        justifyContent: "space-between",
+      } }
+    >
+      <Box
+        sx={ {
+          alignItems: "center",
+          display: "flex",
+          flexDirection: "column",
+          textAlign: "center",
+        } }
+      >
+        <Typography maxWidth={ 800 } mt={ 4 } variant="h1">
+          Check your inbox
+        </Typography>
+
+        <GradientTypography
+          id="verificationLabel"
+          mb={ 4 }
+          style={ { ...theme.typography.emphasized } }
+          variant="h1"
+        >
+          Paste your verification code here.
+        </GradientTypography>
+
+        <TextInputField
+          aria-labelledby="verificationLabel"
+          name="authCode"
+          placeholder="Verification Code"
+          type="text"
+        />
+
+        <Button sx={ { mt: 2 } } type="submit">
+          Reset password
+        </Button>
+      </Box>
+
+      <Box my={ 4 } textAlign="center">
+        { showResendLink ? (
+          <Typography color="grey100" fontWeight={ 500 }>
+            Didn&apos;t received the email?
```

Comment:
Just small typo from copy 
`Didn't receive an email? Click here to resend`

Reply:
Also the same on the Verify page for new users on `pages/signUp/Verification.tsx` if you make the change now instead of adding it to the ticket of copy changes before finishing up MVP.

Reply:
Good call, i fixed this in the latest push.